vllm/worker/hpu_enc_dec_model_runner.py

```diff
@@ -364,14 +364,14 @@ def profile_run(self) -> None:
                              False)
         return
 
-    def warmup_scenario(self,
-                        batch_size,
-                        seq_len,
-                        is_prompt,
-                        kv_caches,
-                        is_pt_profiler_run=False,
-                        is_lora_profile_run=False,
-                        temperature=0) -> None:
+    def warmup_scenario(  # type: ignore[override]
+        self,
+        batch_size,
+        seq_len,
+        is_prompt,
+        kv_caches,
+        is_pt_profiler_run=False,
+    ) -> None:
```

Comment:
why omni need change this API interface?

Reply:
This change is from https://github.com/HabanaAI/vllm-fork/pull/1163/commits/d6f0e9518c17f51c0306a32978a885d9a878c7e6 and https://github.com/HabanaAI/vllm-fork/pull/1163/commits/f167a3cd3a6a3ea8800a715a7965628de12c5071. I think the author made these changes to remove unused variable and for pre-commit code formatting.